### ISSUE TITLE
Add issue templates

### ISSUE THREAD

**Issue by ZeRego** [lightdash/lightdash#235]
We should add issue templates to help all contributors to raise better issue tickets. 

Docs:
[About issue and pull request templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates)
[Configuring issue templates for your repository](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)

**Comment by ZeRego** [comment]
Idea on how it could look like.
```
Closes: #<github issue id>

Description:
<text>

Preview:
<gif/image/video>

Change scope:
[] Frontend
[] Backend
[] Common
[] E2E tests
[] Docs
[] CI/CD
Todo:
[] update Changelog
[] ...
```

**Comment by TuringLovesDeathMetal** [comment]
closed by #955 and #953
